subsys/bluetooth/common/Kconfig

```diff
@@ -7,7 +7,7 @@
 #
 
 config BT_HAS_HCI_VS
-	bool
+	bool "Bluetooth controller supports the Zephyr HCI Vendor-Specific Commands and Event" if !BT_CTLR
```

Comment:
Ad 1) Yeah i know the name is confusing, it's kinda like saying it is "vendor-specific HCI vendor-specific events". Normally as a vendor you would have complete control over the Host/Controller and you don't need this abstraction, but in our case we have some proprietary logic in our controller implementation that we need to expose to the application.

Ad 2) This is a change i don't really like, but it was necessary for me to make the Kconfig symbol visible for the test i wrote. If a Kconfig symbol does not have a prompt you cannot set it in a prj.conf file. Normally the controller Kconfig would set this symbol, but as the test replaces the controller with a test stub, the symbol is no longer set and all the logic for vendor-specific events is compiled out. The reason for the !BT_CTRL is to make the prompt optional; so you can only manually set it if BT_CTRL is not set. Otherwise the controller determines the value.